packages/transaction-controller/src/TransactionController.ts

```diff
@@ -944,6 +945,33 @@ export class TransactionController extends BaseController<
     this.update({ transactions: this.trimTransactionsForState(transactions) });
   }
 
+  /**
+   * Add params to transactionMeta.
+   *
+   * @param transactionMetaId - ID of the transaction.
+   * @param securityAlertResponse - New params to be added to tarnsaction meta.
+   */
+  updateSecurityAlertResponse(
+    transactionMetaId: string,
```

Comment:
Shall we call this `transactionId` as to the caller, we refer to them just as transaction IDs?